src/thor/timedep_reverse.cc

```diff
@@ -43,6 +46,7 @@ void TimeDepReverse::Init(const midgard::PointLL& origll, const midgard::PointLL
   // Reserve size for edge labels - do this here rather than in constructor so
   // to limit how much extra memory is used for persistent objects.
   // TODO - reserve based on estimate based on distance and route type.
+  edgelabels_rev_.clear();
```

Comment:
I ran into issues with growing memory consumption for the reverse search queries. Turns out there was a memory leak here.